infrastructure/utils/scripts/installation/prerequisites/install-docker.sh

```diff
@@ -1,15 +1,6 @@
-#!/bin/bash
+#! /bin/sh
```

Comment:
env too ?

Reply:
`sh` is guaranteed to be located at `/bin/sh`, `bash` is not